cmd/maya-apiserver/cstor-operator/cspc/operations.go

```diff
@@ -58,11 +59,11 @@ func (pc *PoolConfig) expandPool() error {
 
 		// Pool expansion for raid group types other than striped
 		if len(pool.RaidGroups) > len(cspObj.Spec.RaidGroups) {
-			cspObj = addGroupToPool(&pool, cspObj)
+			cspObj = pc.addGroupToPool(&pool, cspObj)
```

Comment:
another comment not related to the code here.. this is related to the fn `syncCSPC` that calls `pc.handleOperations()`
does it makes sense to continue, i.e., reach till `pc.handleOperations()` even if pc.create or GetCSPWithoutDeployment or pc.DownScalePool returns error?
Basically, my qn is: even if one particular day-2 op fails, we can try for other operations that user asked for

Reply:
with above comments, I'm done reviewing `https://github.com/openebs/maya/pull/1358/`, except `DownScalePool`.. I will review `DownScalePool` once I complete other PRs as well.. thanks @sonasingh46 :+1: